### ISSUE TITLE
revenue, net_income and other properties of info_box are wrongly parsed when they follow USD template without space

### ISSUE THREAD

**Issue by nico008** [spencermountain/wtf_wikipedia#375]
Hi spencermountain,
It's been a while!
I believe I just found a problem on version 8.3:
In some cases, the parser will return USD.627 billion instead of USD115.627 billion.
Example:
https://en.wikipedia.org/w/index.php?title=JPMorgan_Chase&action=edit
Could it be because there is no space between the USD template and the number?
Example:
| revenue             = {{increase}} [[United States dollar|US$]]115.627 [[Billion (short scale)|billion]]
I suspect this problem also apply to other templates, potentially out of the Infobox as well.
I would really appreciate if this could be fixed.
Thanks a lot!
Nicolas

**Comment by spencermountain** [comment]
hey @nico008 good find.
yup, can reproduce:
```js
let doc = wtf(`[[US]]9999.2`)
console.log(doc.text())
// 'US.2'
```
will take a look, thanks.

**Comment by nico008** [comment]
Thank you! Could you give an ETA when a new version would be available?
This would help a lot!

**Comment by spencermountain** [comment]
next week

**Comment by spencermountain** [comment]
got this on my list to do tomorrow
thanks for your patience

**Comment by spencermountain** [comment]
released as 8.4.0
cheers

**Comment by nico008** [comment]
Hi @spencermountain,
I updated to version 8.4.0 and I can confirm your fix is working good !
Thank you so much for quick release.
Be well!
